src/utils/function/clamp/clamp.ts

```
@@ -0,0 +1,51 @@
+  if (
+    typeof value !== "number" ||
+    typeof min !== "number" ||
+    typeof max !== "number"
+  ) {
+    errorHandler.throwTypeError(
+      "number",
+      `typeof value: ${typeof value}, min: ${typeof min}, max: ${typeof max}`
+    );
+  }
+
+  if (isNaN(value) || isNaN(min) || isNaN(max)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)